config/crd/kustomization.yaml

```diff
@@ -5,9 +5,10 @@ commonLabels:
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
-- bases/infrastructure.cluster.x-k8s.io_virtinkclusters.yaml
-- bases/infrastructure.cluster.x-k8s.io_virtinkmachines.yaml
-- bases/infrastructure.cluster.x-k8s.io_virtinkmachinetemplates.yaml
+  - bases/infrastructure.cluster.x-k8s.io_virtinkclusters.yaml
+  - bases/infrastructure.cluster.x-k8s.io_virtinkclustertemplates.yaml
+  - bases/infrastructure.cluster.x-k8s.io_virtinkmachines.yaml
+  - bases/infrastructure.cluster.x-k8s.io_virtinkmachinetemplates.yaml
```

Comment:
Just add the new CRD to this list, don't format this file

Reply:
it's formatted by vscode, maybe I need set it.